go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -3558,7 +3569,19 @@ func (fbo *folderBranchOps) finalizeMDWriteLocked(ctx context.Context,
 		}
 	}
 
-	md.loadCachedBlockChanges(ctx, bps, fbo.log)
+	md, copied := md.loadCachedBlockChanges(ctx, bps, fbo.log,
```

Comment:
Can we make a `folderBranchOps` helper function for this repeated block of code?  Not sure if `md` or `irmd` has to live on after the function is called, but if so it could return them.